web-rdfs/source/http_handlers.cc

```diff
@@ -101,11 +101,34 @@ std::string get_group(std::shared_ptr<HttpsServer::Request> request) {
 }
 
 void create_file_handler(std::shared_ptr<HttpsServer::Response> response,
-                         std::shared_ptr<HttpsServer::Request> request) {
+                         std::string path) {
   LOG(DEBUG) << "HTTP request: create_file_handler";
 
-  // TODO(security): implement
-  response->write("create_file_handler");
+  std::string input = "hdfs dfs -fs hdfs://localhost:5351 -put " + path;
+
+  hadoop::hdfs::CreateResponseProto res;
+  hadoop::hdfs::CreateRequestProto req;
+
+  req.set_src(path);
+
+  zkclient::ZkNnClient::CreateResponse zkResp = zk->create_file(req, res);
+
+  *response << webRequestTranslator::getCreateResponse(path);
```

Comment:
can change this to response->write(SimpleWeb::StatusCode:: success_created); if CreateResponse is ok otherwise is can be response->write(SimpleWeb::StatusCode:: server_error_internal_server_error);